packages/block-library/src/group/edit.js

```diff
@@ -68,6 +90,12 @@ function GroupEdit( { attributes, setAttributes, clientId } ) {
 
 	return (
 		<>
+			<BlockControls>
+				<BlockVerticalAlignmentControl
```

Comment:
Is there any reason for this to be specific to the group block? Feels like something generic that can be used by any "flex" layout. (So we can move its implementation details both edit/save to `flex.js` file).